pkg/apis/application/v1alpha1/types.go

```diff
@@ -312,6 +312,8 @@ type ApplicationSourceKustomize struct {
 	Images KustomizeImages `json:"images,omitempty" protobuf:"bytes,3,opt,name=images"`
 	// CommonLabels adds additional kustomize commonLabels
 	CommonLabels map[string]string `json:"commonLabels,omitempty" protobuf:"bytes,4,opt,name=commonLabels"`
+	// CommonAnnotations adds additional kustomize commonAnnotations
+	CommonAnnotations map[string]string `json:"commonAnnotations,omitempty" protobuf:"bytes,6,opt,name=commonAnnotations"`
```

Comment:
Could you please move that down to the bottom of the struct, so that its number matches the order in the struct? This makes it less error prone on further additions to `ApplicationSourceKustomize` (people tend to look at the last entry's number and increase that when adding new entries)